xml/depl_require.xml

```diff
@@ -1406,6 +1406,110 @@
     For storage requirements, see <xref linkend="sec.depl.req.storage.hardware.store"/>.
    </para>
   </sect2>
+
+<sect2 xml:id="sec.depl.req.hardware.monnode">
+   <title>&monnode;</title>
+   <para>
+     The Monasca agent roles, such as <literal>monasca-agent</literal> and
```

Comment:
I think the information where to deploy which roles rather belongs to the Monasca barclamp section. The only piece of information I would mention here is probably the following: 
The &monnode; is a dedicated physical machine that runs the ``monasca-server``` role. It is used for &productname; Monitoring.  

In fact, even this definition (what the Monasca node is and what is it good for), should be covered in chapter 1 already (I will add it there as part of bsc#1042220 as soon as I have received some input by Johannes).